src/sentry/static/sentry/app/views/app/index.tsx

```diff
@@ -178,13 +182,16 @@ class App extends React.Component {
     OrganizationsStore.load([]);
   }
 
+  mainContainerRef = React.createRef<HTMLDivElement>();
+
   updateTracing() {
     const route = getRouteStringFromRoutes(this.props.routes);
     setTransactionName(route);
   }
 
   handleConfigStoreChange(config) {
-    const newState = {};
+    // TODO(ts): What's the best way to type this?
+    const newState: any = {};
```

Comment:
Here's a playground if you want to suggest something, but I'm not going to spend time on this: https://www.typescriptlang.org/play/#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wG4AocmATzCTgGUYUZ6BeOAb3LjgBsIKACbAAdgHMAXHABGECH1SiKvJFBxRpchUpVxRSJEIDOAVTDiowpFvmKUynvqQB3Y4pisoAYQijjSKIwAAo44DC2Og56AK4BUAD80pz6KCA2cMYwUGLicAC+FIWUaHwoxsZwAGLycEgAHqyiJojEMAB0vuB+gTAAPJz5ADSMzKwAfFyUvFks7FO8vALCudKYKHwBQ06q6tBrG1s7zkZmFlZCGeubSNuLzm4eXr7+vaG4YBFw10e8xU4ACwcQkUL0wwHETGgSG8QIkSAAFGg-OCpHAHNQAJQLRYAelxcAAKgB5AAixIRMGMmOkAHUgTB8JUYAD6DIkFk4C4UNQ4DAIHzaPQWcBjAljsj-PADC4mHNpMEULBgBs+nKJnAOIM9LxgJg4EiURD2gZTuZLNY4ABCNgcGLNJDggxCbHce68GXqpAmwwmc0XeaS1E+s3naw6grHPUGoPGuJqa22uD2y5Ooyu44e1xe9rxqCauCx8S5+IR-JR-UI4kyABWSAw7QA1khqMYEZ6xkhMe1FBIWXBJgAGDPuvkA0XtAIwL3t7OdzFlpzl3hEB1QBEjxZEGAxKCifQxPh8PTl-JAA

Reply:
I think this should work:
```suggestion
    const newState = {} as State;
```

Reply:
Yea that works :D

Reply:
Ahh awesome, thanks!

Reply:
You're welcome 😊